packages/core/src/application.ts

```diff
@@ -41,6 +41,12 @@ export class Application extends Context {
         this.controller(ctor);
       }
     }
+
+    if (options.models) {
```

Comment:
Based on https://github.com/strongloop/loopback-next/issues/742, should we remove `options.models` support?

Reply:
ah yes, i wrote this before the discussion. I'll get rid of this option